code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -278,13 +278,12 @@
 
 /obj/item/grenade/chem_grenade/teargas/Initialize()
 	. = ..()
-	var/obj/item/reagent_containers/glass/beaker/large/B1 = new(src)
-	var/obj/item/reagent_containers/glass/beaker/large/B2 = new(src)
-	B1.reagents.add_reagent(/datum/reagent/phosphorus, 40)
-	B1.reagents.add_reagent(/datum/reagent/potassium, 40)
-	B1.reagents.add_reagent(/datum/reagent/capsaicin/condensed, 40)
-	B2.reagents.add_reagent(/datum/reagent/sugar, 40)
-	B2.reagents.add_reagent(/datum/reagent/capsaicin/condensed, 80)
+	var/obj/item/reagent_containers/glass/beaker/B1 = new(src)
+	var/obj/item/reagent_containers/glass/beaker/B2 = new(src)
+	B1.reagents.add_reagent(/datum/reagent/capsaicin, 30)
```

Comment:
what is the difference mechanically between capsaicin and capsaicin/condensed?

Reply:
condensed is what is supposed to be in a grenade and teargas, normal is what is in food and just gives taste and pain if you eat too much of it.